addon/src/components/animated-if.ts

```diff
@@ -5,27 +5,29 @@ import { computed } from '@ember/object';
   A drop in replacement for `{{#if}}` that animates changes when the predicate changes.
   Animated-if uses the same arguments as animated-each.
   ```hbs
-  <button {{action toggleThing}}>Toggle</button>
+  <button {{on 'click' this.toggleThing}}>Toggle</button>
 
   {{#animated-if showThing use=transition}}
-      <div class="message" {{action "toggleThing"}}>
-          myContent
-      </div>
+    <div class="message" {{on 'click' "toggleThing"}} role="button">
```

Comment:
@janwerkhoven this should actually become 

```suggestion
    <div class="message" {{on 'click' this.toggleThing}} role="button">
```